src/pages/auth/Auth.tsx

```diff
@@ -23,6 +23,17 @@ const Auth: React.FC<ContextProps> = ({ user }): React.ReactElement => {
     }
   }
 
+  const queryString = window.location.search;
```

Comment:
The `window.location.search` is not reactive. We need the `useLocation` hook here.